wurf_android_copy_lib_to_app.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""Tool for copying jni libraries to the appropriate Android app folder."""
+
+import os
+import shutil
+from waflib.TaskGen import feature
+from waflib.TaskGen import after_method
+
+from waflib import Task, Errors, Logs
+
+
+@feature('android_copy_lib_to_app')
+@after_method('apply_link')
+def android_copy_lib_to_app(self):
+    """
+    Copy library to Android application.
+
+    When building an Android application from the IDE native libraries placed
+    in the libs folder will be packaged in the apk. So we copy the shared libs
+    we build there.
+    """
+    if not self.bld.is_mkspec_platform('android'):
+        return
+
+    input_librarys = self.link_task.outputs
```

Comment:
libraries ;)
Actually, I like input_libs better! We should not overuse the no-abbreviation policy, where the meaning is absolutely clear.